gapic/templates/setup.py.j2

```diff
@@ -46,12 +48,14 @@ dependencies = [
     {% endif %}
     {% endfor %}
 ]
+{% if rest_async_io_enabled %}
 extras = {
     "async_rest": [
         "google-api-core[grpc] >= 2.20.0, < 3.0.0dev",
         "google-auth[aiohttp] >= 2.35.0, <3.0.0dev"
     ],
 }
+{% endif %}
```

Comment:
I've addressed this but I personally preferred the previous solution only because that didn't update the setup file in every GAPIC with an empty extra.

Reply:
Either way is fine. This makes it explicit there are no extras, on purpose.